wikimedia/wikipedia-ios
# Update Watchlist feature initialization

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Updates `WKAppEnvironment` theme propagation and the initialization of the Watchlist feature. Depends on merge of https://github.com/wikimedia/wikipedia-ios-components/pull/24.

### Test Steps
1. Use staging build or update `FeatureFlags` to enable Watchlist
2. Add items to your Watchlist
3. Open Watchlist
4. Confirm the Watchlist renders
5. Catch theme, return to Watchlist and confirm new theme is rendered
